CHANGELOG.md

```diff
@@ -467,12 +467,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - _WD_DownloadFile
 - Global variable to hold session details
```

Comment:
reformating proposal
`- _WD_DownloadFile: Global variable to hold session details`

Reply:
Aha. These are two separate changes.
Ok.